src/logic/basic.lean

```diff
@@ -196,8 +196,8 @@ variables {a b c d : Prop}
 
 /-! ### Declarations about `implies` -/
 
-instance : is_refl Prop iff := ⟨iff.refl⟩
-instance : is_trans Prop iff := ⟨λ _ _ _, iff.trans⟩
+instance Prop.is_refl : is_refl Prop iff := ⟨iff.refl⟩
+instance Prop.is_trans : is_trans Prop iff := ⟨λ _ _ _, iff.trans⟩
```

Comment:
What was the auto-generated name here?

Reply:
This one should be called `iff.is_trans` ideally,

Reply:
it actually got it right, there was no need for this! overzealous, ty